src/thunderbots/software/ai/passing/pass.cpp

```diff
@@ -24,6 +26,11 @@ Point Pass::receiverPoint() const
     return receiver_point;
 }
 
+Angle Pass::receiverAngle() const
+{
+    return vertexAngle({0, 0}, receiverPoint(), passerPoint());
```

Comment:
If this is supposed to return the angle/orientation the receiver should face, you can do `(passerPoint() - receiverPoint()).orientation()`

Reply:
much better, thanks. changed